### ISSUE TITLE
commonmeta doesn't include member ids

### ISSUE THREAD

**Issue by gbilder** [front-matter/commonmeta#3]
Steps to reproduce:

```bash
cd /tmp
git clone https://github.com/front-matter/commonmeta.git
cd commonmeta
go build -o cm 
./cm con
./cm convert "10.5555/12345678" --from crossref 2> cm-out.json
```

View 'cm-out.json'

**Comment by mfenner** [comment]
The REST API provides a `member` key, "7822" in this case: https://api.crossref.org/works/10.5555/12345678

**Comment by mfenner** [comment]
Fixed via 94914e9 and released [v0.41](https://github.com/front-matter/commonmeta/releases/tag/v0.4.1) (also upgrading to go 1.23). If you have cloned the github repo, you can do

```bash
go run main.go convert "10.5555/12345678" --from crossref
```
and will see

```json
"publisher": {
    "id": "https://api.crossref.org/members/7822",
    "name": "Test accounts"
  },
```
